lib/iris/experimental/regrid.py

```diff
@@ -435,31 +421,12 @@ def _regrid_area_weighted_array(
         The X dimension within `src_data`.
     * y_dim:
         The Y dimension within `src_data`.
-    * src_x_bounds:
-        A NumPy array of bounds along the X axis defining the source grid.
-    * src_y_bounds:
-        A NumPy array of bounds along the Y axis defining the source grid.
-    * grid_x_bounds:
-        A NumPy array of bounds along the X axis defining the new grid.
-    * grid_y_bounds:
-        A NumPy array of bounds along the Y axis defining the new grid.
-    * grid_x_decreasing:
-        Boolean indicating whether the X coordinate of the new grid is
-        in descending order.
-    * grid_y_decreasing:
-        Boolean indicating whether the Y coordinate of the new grid is
-        in descending order.
-    * area_func:
-        A function that returns an (p, q) array of weights given an (p, 2)
-        shaped array of Y bounds and an (q, 2) shaped array of X bounds.
-    * weights_info: The area weights to be used for area-weighted regridding.
+    * weights_info:
+        The area weights information to be used for area-weighted
+        regridding.
```

Comment:
This information is now in the _calculate_regrid_area_weighted_weights function.